tests/test_cli.py

```diff
@@ -482,3 +482,55 @@ def test_command_blobs_antialiased(self):
         )
         masked = encode_base64_image(image_path('map_noblobs_antialiased.png'))
         assert masked == result.output.strip()
+
+    def test_binarize(self):
+        result = self.runner.invoke(
+            cli.binarize,
+            [self.image_map],
+        )
+        binarized = encode_base64_image(image_path('map_bw.png'))
+        assert binarized == result.output.strip()
+
+    def test_binarize_default(self):
```

Comment:
Same as in the previous comment. Since the default is Li, if I'm not mistaken, renaming one of these two tests might clarify the intent/difference.